client/containers/PipelineContainer.tsx

```diff
@@ -12,10 +13,15 @@ export default class PipelineContainer extends ChildContainer<
     super();
     this.state = {
       currentPartner: 'ALL',
+      currentTeam: 'Boston', // 2/ can I use cuc here to do cuc.user.team, or how do I get the current user's current team?
```

Comment:
2/ can I use cuc here to do `cuc.user.team`? (in which case I'd be importing CurrentUserContainer at the top), or how do I get the current user's current team?

Reply:
Use cuc here. You should import as necessary

Reply:
Is this how I'd access `cuc.user` here? Not sure if correct – I see async's used to get userId's in the other container tsx files, nothing like as below

```
import { CurrentUserContainer } from '../../containers/CurrentUserContainer';
```
at the top, and then for the highlighted line here:

currentTeam =  <Subscribe to={[CurrentUserContainer]}>
                            {(cuc: CurrentUserContainer) => (cuc.user.team)}
                          </Subscribe>
```

Reply:
`cuc.user` via a subscribe component is generally correct. You'll need to use `partnerTeam` though: https://github.com/dormroomfund/magic-wand/blob/master/client/schemas/user.ts#L39

Reply:
Don't think makes sense here - I tried and got a memory leak  issue re: Can't perform a React state update on an unmounted component. If there's a quick soln to this I missed pls lmk!

Didn't see cuc called in any of the other containers besides the currentUserContainer itself, so taking a diff approach

Reply:
Oh we're in a container, my bad. Every container has access to every other container through the `ctx` property.